pkgs/development/interpreters/clojure/nbb/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, fetchurl
+, babashka
+, cacert
+, clojure
+, git
+, jdk
+, nbb
+, nodejs
+, fetchFromGitHub
+, makeWrapper
+, runCommand }:
+
+stdenv.mkDerivation rec {
+  pname = "nbb";
+  version = "0.5.103";
+
+  src = fetchFromGitHub {
+    owner = "babashka";
+    repo = pname;
+    rev = "release_v${version}";
+    sha256 = "2H+mDleLp3PQCByRDJtSnuBf7dRhnZAUBOdQ/vHsYgE=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [ babashka cacert git jdk nodejs ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    mkdir -p .m2
+    substituteInPlace deps.edn --replace ':paths' ':mvn/local-repo "./.m2" :paths'
+    substituteInPlace bb.edn --replace ':paths' ':mvn/local-repo "./.m2" :paths'
+    echo deps.edn
+
+    runHook postConfigure
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    export DEPS_CLJ_TOOLS_DIR=${clojure}
+    export DEPS_CLJ_TOOLS_VERSION=${clojure.version}
+    mkdir -p .gitlibs
+    mkdir -p .cpcache
+    export GITLIBS=.gitlibs
+    export CLJ_CACHE=.cpcache
+
+    mkdir -pv $out/lib/node_modules/nbb
+    bb release
```

Comment:
? 
where's the line? I don't know what you're talking about. -j ?
And, the current code was slightly modified. (The buildPhase has been slightly changed to enable loading of node packages connected to node_path. i'll commit that.)
.
However, I have one question. I want to give NODE_PATH as a prefix when makeWrapper in installPhase in advance. Is there a way to get all the NODE_PATHs installed in the system at once? 

(I'm not know about this method because I only use node2nix, but I want to make all packages like Nixpkgs basic nodePackages.(...) installed in the system available for import.

                    makeWrapper '${pkgs.nodejs}/bin/node' "$out/bin/nbb" \
                    --prefix NODE_PATH : ${NODE_PATH} \ <-- how to reference (installed all nodePackages as nix?
                    --add-flags "$out/lib/nbb_main.js"

Reply:
`-j2` is usually used to indicate that we want to build in parallel.

I think NODE_PATH only exists in bash so you can't reference it with nix.